protect_archiver/config.py

```diff
@@ -1,12 +1,15 @@
+from typing import Optional
+
+
 class Config:
-    def __init__(self):
+    def __init__(self) -> None:
         pass
 
     ADDRESS: str = "unifi"
     PORT: int = 443
     PROTOCOL: str = "https"
     USERNAME: str = "ubnt"
-    PASSWORD: str = None
+    PASSWORD: Optional[str] = None
```

Comment:
Same here. I don't think any of the authentication variables can ever be considered optional.

Reply:
same as above